params/history_segment.go

```diff
@@ -0,0 +1,249 @@
1663465363661316364223a7b22696e6465783a6f6d6974656d707479223a32312c22766f74655f61646472657373223a5b302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c302c305d7d7d2c22726563656e7473223a7b223331323638353839223a22307838623663386664393364366634636561343262626233343564626336663064666462356265633733222c223331323638353930223a22307861366637396236303335396631343164663930613063373435313235623133316361616666643132222c223331323638353931223a22307862323138633564366166316639373961633432626336386439386135613064373936633661623031222c223331323638353932223a22307862346464363664376332633765353766363238323130313837313932666238396434623939646434222c223331323638353933223a22307862653830376464646230373436333963643966613631623437363736633036346663353064363263222c223331323638353934223a22307863633865366430306331376562343331333530633663353064386238663035313736623930623131222c223331323638353935223a22307863653266643735343465306232636339343639326434613730346465626566376263623631333238222c223331323638353936223a22307865326433613733396566666364336139393338376430313565323630656566616337326562656131222c223331323638353937223a22307865396165333236316134373561323762623130323866313430626332613763383433333138616664222c223331323638353938223a22307865613061366533633531316262643130663435313965636533376463323438383765313162353564222c223331323638353939223a22307865653232363337396462383363666663363831343935373330633131666464653739626134633063227d2c22726563656e745f666f726b5f686173686573223a7b223331323638353739223a226231396466346132222c223331323638353830223a226231396466346132222c223331323638353831223a226231396466346132222c223331323638353832223a226231396466346132222c223331323638353833223a226231396466346132222c223331323638353834223a226231396466346132222c223331323638353835223a226231396466346132222c223331323638353836223a226231396466346132222c223331323638353837223a226231396466346132222c223331323638353838223a226231396466346132222c223331323638353839223a226231396466346132222c223331323638353930223a226231396466346132222c223331323638353931223a226231396466346132222c223331323638353932223a226231396466346132222c223331323638353933223a226231396466346132222c223331323638353934223a226231396466346132222c223331323638353935223a226231396466346132222c223331323638353936223a226231396466346132222c223331323638353937223a226231396466346132222c223331323638353938223a226231396466346132222c223331323638353939223a226231396466346132227d2c226174746573746174696f6e3a6f6d6974656d707479223a7b22536f757263654e756d626572223a33313236383539372c22536f7572636548617368223a22307863653762343062643836353463306339653961653131373063393839666531353138323562356235306163313063313838326539383331366164643362343833222c225461726765744e756d626572223a33313236383539382c2254617267657448617368223a22307837323335343633626530633139623361626461366666653766393035633936613861643632616565393663616239396465613937303531626433346564313366227d7d"
```

Comment:
Hard code is key for history segment feature, nodes could follow the history prune rule as same, it's also a simplest way.  And user can use custom history segment config in private chain.

Reply:
Can't we make it so that these bytes are read from a file at least?

Reply:
If these data read from file, it will cause more maintenance resource. I think hard code is ok, like `core/systemcontracts/upgrade.go` does.

Reply:
The pattern in `upgrade.go` is not great either, because it crashes the IDE loading all the data.  If you can't think of any better way, at least one thing we can do is keep this data in a separate .go file so that the struct definitions below are easier to locate.